app/src/main/java/com/android/voyageur/ui/components/UserProfileContent.kt

```diff
@@ -136,7 +145,7 @@ fun UserProfileContent(
                     else -> onAddContact?.invoke()
                   }
                 },
-                enabled = true, // Allow the button to be clickable in all states
+                enabled = isConnected, // Allow the button to be clickable in all states
```

Comment:
Here when you change something, could you please either update or remove the old comments? It becomes confusing otherwise.